Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14-alpine
+FROM node:16-alpine
 
 COPY . .
 
```

Comment:
Install the correct version of NPM here.  Currently this dockerfile uses whatever comes with npm.